docs/guides/dynamo_run.md

```diff
@@ -125,15 +133,14 @@ source $HOME/.cargo/env
 
 #### Step 3: Build
 
-Run `cargo build` to install the `dynamo-run` binary in `target/debug`.
+Run `cargo build --features cuda -p dynamo-run` to install the `dynamo-run` binary in `target/debug`.
```

Comment:
I think this step is intended to be runnable on macos, and there's a specific shout out for building with cuda support on linux+gpu below -- do we need `--features cuda` in this spot?